module-2/backend/week_four.md

```diff
@@ -10,14 +10,44 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+  - A cookie is a small file that lives on the client end. It is generated in the response cycle. It helps create the impression of "state" by passing back relevant information in the request process. This can help the server remember who we are and what we were up to last time our devices talked to eachother.
+  
 * What’s the difference between a session and a cookie?
+    - A cookie is is stored in the server/locally.
+    - A session is stored in the browser.
```

Comment:
As we've been using them, both cookie and session are stored by the client (browser). The sessions we've been using are _encrypted_ portions of the cookie. Information in a session can be held on the server, but do not believe that's default Rails behavior.